build-logic/kotlin/build.gradle.kts

```diff
@@ -8,5 +8,8 @@ dependencies {
     implementation(projects.gradleExt)
     implementation(projects.testing)
     implementation(libs.kotlinGradle)
+    implementation(libs.kotlinx.serialization.gradle)
     implementation(libs.ksp)
+    // workaround for https://github.com/gradle/gradle/issues/15383
+    implementation(files(libs.javaClass.superclass.protectionDomain.codeSource.location))
```

Comment:
oh my god

Reply:
It's an old trick, look at all the neighbor modules; still waiting for out-of-the-box solution